server/events/project_command_builder_test.go

```diff
@@ -875,7 +876,29 @@ projects:
 `
 
 	exactSymbols := []string{"", "="}
-	nonExactSymbols := []string{">", ">=", "<", "<=", "~="}
+	// Depending on when the tests are run, the > and >= matching versions will have to be increased.
+	// It's probably not worth testing the terraform-switcher version here so we only test <, <=, and ~>.
+	// One way to test this in the future is to mock tfswitcher.GetTFList() to return the highest
+	// version of 1.3.5.
+	// nonExactSymbols := []string{">", ">=", "<", "<=", "~>"}
+	nonExactSymbols := []string{"<", "<=", "~>"}
+	nonExactVersions := map[string]map[string][]int{
+		// ">": {
+		// 	"project1": {1, 3, 5},
+		// },
+		// ">=": {
+		// 	"project1": {1, 3, 5},
+		// },
```

Comment:
can this be deleted?

Reply:
It could but I figured we could leave it for posterity until we could possibly mock it ?

Reply:
sure, that is ok

Reply:
Thank you!